load.php

```diff
@@ -5,13 +5,15 @@
 use HM\Platform;
 
 require_once __DIR__ . '/inc/namespace.php';
+require_once __DIR__ . '/inc/passwords/namespace.php';
```

Comment:
The `namespace.php` thing still weirds me out. For passwords, I get it maybe, but `inc/namespace.php` IMO makes zero sense.